package.json

```
@@ -0,0 +1,103 @@
+{
+  "name": "track-mail-2019-a-opalev",
+  "homepage": "https://chexex.github.io/cra-caddy-docker/",
```

Нужно поменять на свой :)

```suggestion
  "homepage": "https://FilippovIvan19.github.io/2019-2-Track-Frontend-I-Filippov/",
```